src/box/lua/config/configdata.lua

```diff
@@ -246,6 +253,177 @@ local function effective_snapshot_dir(iconfig)
     return fio.abspath(fio.pathjoin(work_dir, snap_dir))
 end
 
+-- Read snap file and return a map of saved UUIDs and names for
+-- all instances and for the current repicaset.
+local function snapshot_names(snap_path)
+    -- Hardcoded space IDs, used to determine the names, persisted
+    -- in snapshot. Don't determine IDs dynamically, as it's needed
+    -- to read xlog file twice: getting IDs from _space and only after
+    -- that getting names from _schema and _cluster. IDs are anyway
+    -- hardcoded in box/schema_def.h.
+    local space_ids = {
+        ['_schema']  = 272,
+        ['_cluster'] = 320,
```

Comment:
There are
```
tarantool> box.schema.SCHEMA_ID
---
- 272
...

tarantool> box.schema.SPACE_ID
---
- 280
...


```
For it. But unfortunately they are exposed only after `box.cfg` call.
I think you can expose these constants even before box.cfg{} and use them.

Reply:
Great idea. Exposed `box.schema` before box.cfg